src/Config.php

```diff
@@ -5,9 +5,12 @@
 use Noodlehaus\Exception\FileNotFoundException;
 use Noodlehaus\Exception\UnsupportedFormatException;
 use Noodlehaus\Exception\EmptyDirectoryException;
+use InvalidArgumentException;
+use Noodlehaus\FileParser\FileParserInterface;
+use Noodlehaus\StringParser\StringParserInterface;
 
 /**
- * Config
+ * Configuration reader and writer for PHP.
```

Comment:
Please, add yourself as author in every file you edited.

Reply:
I've fixed this.